contracts/access/AccessControl.sol

```diff
@@ -48,11 +48,11 @@ import {ERC165} from "../utils/introspection/ERC165.sol";
  */
 abstract contract AccessControl is Context, IAccessControl, ERC165 {
     struct RoleData {
-        mapping(address => bool) members;
+        mapping(address account => bool isMember) members;
         bytes32 adminRole;
     }
 
-    mapping(bytes32 => RoleData) private _roles;
+    mapping(bytes32 role => RoleData data) private _roles;
```

Comment:
Here I don't think the `data` name is necessary because it's implicit from the `RoleData` struct name.

Reply:
If I may leave my opinion here, I don't think any of the last names are needed because they are a duplicate of the name of the whole mapping

Reply:
Yeah I think that is probably generally true.

Reply:
I'm ok with removing the `data` from `RoleData` but not sure about generalizing. Look at these cases:

```
// ERC20
mapping(address account => mapping(address spender => uint256 value)) private _allowances;

// AccessControl
```solidity
mapping(address account => bool isMember) members;
```

The ERC20 case is one where the mapping is nested. Because there's no name for the nested `address => value` mapping, it's no obvious.

In the AccessControl, the mapping name is not explicitly representative of what it is, so removing `isMember` would be detrimental imo

Reply:
Just finishing reviewing and I think most cases are like @RenanSouza2 is describing, so I'd agree with also removing those names if we agree. Aside from cases like the ones I shared, the guideline would be to avoid naming the `value` side of the mapping if its name is equal or very similar to the mapping's own name.

Reply:
I agree with removing.

Reply:
Just removed, I renamed a few of them where I see it makes sense. Regarding my counterexamples, this is what I did:

```solidity
// Previously
// mapping(address account => mapping(address spender => uint256 value)) private _allowances;
mapping(address account => mapping(address spender => uint256)) private _allowances;

// Previously
// mapping(address account => bool isMember) members;
mapping(address account => bool) hasRole;
```